config/config.go

```diff
@@ -334,6 +340,8 @@ type ScrapeConfig struct {
 	ServersetSDConfigs []*ServersetSDConfig `yaml:"serverset_sd_configs,omitempty"`
 	// MarathonSDConfigs is a list of Marathon service discovery configurations.
 	MarathonSDConfigs []*MarathonSDConfig `yaml:"marathon_sd_configs,omitempty"`
+	// MarathonSDConfigs is a list of Marathon service discovery configurations.
+	MesosSDConfigs []*MesosSDConfig `yaml:"mesos_sd_configs,omitempty"`
```

Comment:
This is a bit confusingly named, as it's mesos slave discovery rather than mesos discovery.

Do you not a configuration management system that knows where all your slaves are?